style/scaffold.less

```diff
@@ -45,6 +45,33 @@ body {
 .panel-header {
   margin: 20px auto;
   border-radius: 0;
+  position: relative;
+
+  .task-details{
+    font-size: 0.8em;
+    font-weight: normal;
+    text-transform: uppercase;
+    padding: 6px 12px;
+    position: absolute;
+    right: 0;
+    top: -2em;
+  }
+  button.task-details{
+    cursor: pointer;
+    top: -2.75em;
+  }
+}
+
+.task-details-popover{
+  font-size: 0.8em;
```

Comment:
@Fredasaurus what do you think? what should the styles be for this little guy? I was just guessing at things that look kinda okay.